src/kajol/FindSecondMaximumNumberFromAnArray.java

```diff
@@ -0,0 +1,27 @@
+/*Assignment - 29 : 26th Nov'2020 
+Find second maximum from array without sorting the array.*/
+
+package kajol;
+
+import java.util.Arrays;
+
+public class FindSecondMaximumNumberFromAnArray {
+		void findSecondMaximum(int []inputArray) {
+			System.out.println("Given Array : "+ (Arrays.toString(inputArray)));
+			int firstMaximum=0;
+			int secondMaximum=0;
+			for (int index=0;index<inputArray.length;index++) {
+				if(inputArray[index]>firstMaximum) {
+					secondMaximum=firstMaximum;
+					firstMaximum=inputArray[index];
+				}else if (inputArray[index]>secondMaximum)
```

Comment:
you should one extra condition for max number is like

else if (input[index] > secondMax && input[index] != max)